src/app/components/dropdown/index.html

```diff
@@ -23,6 +23,12 @@
       defaultValue="default">
       Specifies a background color for the dropdown button. If unspecified, the <stache-code>default</stache-code> value sets the background color to <stache-code>sky-color-white</stache-code>. To change the background color to <stache-code>sky-color-blue</stache-code>, specify <stache-code>primary</stache-code>.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="dismissOnBlur"
+      isOptional="true"
+      defaultValue="true">
+      Indicates whether to close the dropdown when users click away from the menu. This property accepts boolean values.
```

Comment:
In the list toolbar docs, the types are formatted with the stache-code element. But that's just because that style was already in place there, and I didn't think to ask Steve whether the types should be formatted. @Blackbaud-SteveBrush, should `boolean` be formatted as a code snippet here? Or should I remove the formatting on the types in https://github.com/blackbaud/skyux2-docs/pull/162?